README.md

```diff
@@ -253,6 +253,7 @@ import DeviceInfo from 'react-native-device-info';
 | [getDeviceType()](#getDeviceType)                 | `string`            |  ✅  |   ✅    |   ❌    | ?      |
 | [isAutoDateAndTime()](#isAutoDateAndTime)         | `boolean`           |  ❌  |   ✅    |   ❌    | 0.29.0 |
 | [isAutoTimeZone()](#isAutoTimeZone)               | `boolean`           |  ❌  |   ✅    |   ❌    | 0.29.0 |
+| [supportedABIs()](#supportedABIs)                 | `string[]`          |  ✅  |   ✅    |   ❌    | 1.0.1  |
```

Comment:
Will be 1.1.0. Use that to update the Changelog.md file too 🙏

Reply:
done. 🙏

Reply:
sorry, has been fixed now.